ci/citest/acceptance-test/multibox/10.0.100.14-vdc-executor-lxc/guestroot/etc/mesos-slave/attributes

```diff
@@ -1 +1 @@
-hypervisor:lxc
+hypervisor:lxc;node-groups:linuxbr
```

Comment:
I don't think it's possible for multiline: See /usr/bin/mesos-init-wrapper.

For comma, scheduler/mesos.go:134